packages/zkdb/package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-plugin-jest": "^27.6.3",
     "eslint-plugin-o1js": "^0.4.0",
     "husky": "^7.0.1",
-    "jest": "^29.7.3",
+    "jest": "^29.7.0",
```

Comment:
Why we need to update jest here?

Reply:
Jest doesn't have version 29.7.3 (https://www.npmjs.com/package/jest?activeTab=versions)
Only the latest stable is 29.7.0 and then 30.0.0-alpha, so when setting jest 29.7.3 and use yarn install, it will show us an interactive console to choose the right version, making the building process error.